daemon/openrazer_daemon/dbus_services/dbus_methods/lanceheadte.py

```diff
@@ -0,0 +1,517 @@
+from openrazer_daemon.dbus_services import endpoint
+
+
+@endpoint('razer.device.lighting.logo', 'setLogoWave', in_sig='i')
+def set_logo_wave_lancehead_te(self, direction):
```

Comment:
Can we remove the `_lancehead_te` suffix on all these methods here? They don't add any value imo (yes, I know that other methods also have such a suffix but I don't like them).

Reply:
Removed as requested